examples/run_language_modeling.py

```diff
@@ -130,9 +130,9 @@ def __init__(self, tokenizer: PreTrainedTokenizer, args, file_path: str, block_s
         logger.info("Creating features from dataset file at %s", file_path)
 
         with open(file_path, encoding="utf-8") as f:
-            lines = [line for line in f.read().splitlines() if len(line) > 0]
+            lines = [line for line in f.read().splitlines() if (len(line) > 0 and not line.isspace())]
```

Comment:
First sub-clause of the if clause is probably covered by the second sub-clause, not?

Reply:
Unfortunately not, as tokenizing the wikitext-2 yields 1 character strings containing a blank space.

Reply:
Ah, I misunderstood your comment but `''.isspace()` returns `False` so both clauses are necessary

Reply:
I see, I was mistaken, thank you